test/packages/false_positives/cisco_asa.expected_errors

```diff
@@ -2,3 +2,14 @@
 <failure>test case failed: one or more problems with fields found in documents: \[0\] parsing field value failed: field &#34;event.type&#34; value &#34;error&#34; is not one of the expected values \(access, allowed, connection, denied, end, info, protocol, start\) for any of the values of &#34;event.category&#34; \(network\)</failure>
 <failure>test case failed: one or more problems with fields found in documents: \[0\] parsing field value failed: field &#34;event.type&#34; value &#34;deletion&#34; is not one of the expected values \(access, allowed, connection, denied, end, info, protocol, start\) for any of the values of &#34;event.category&#34; \(network\)</failure>
 <failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
+<failure>detected ingest pipeline warnings: 2: character class has &#39;-&#39; without escape&#xA;regular expression has redundant nested repeat operator.*</failure>
```

Comment:
There is one failure per pipeline test.

This error is the result of installing the pipelines of `cisco_asa`. Previously, pipelines were installed just onace, now those pipelines are installed once per test (Run()).

Reply:
Will these failures be predictable if we parallelize?

At some point we may consider installing the pipelines by installing the packages, then we could install the package only once. The pipelines we test now are not exactly the same as what is executed at the end, see https://github.com/elastic/elastic-package/issues/1743.

Reply:
Maybe another mid-ground option is to install all pipelines included in the package in the `Setup` step, and remove them on `TearDown`, but then we would be installing all pipelines, even the ones that are not tested. Not sure I like this approach.

Reply:
> Will these failures be predictable if we parallelize?

I guess this would be a source of flakiness. A pipeline test running in parallel with another one could report the errors from the other pipeline test and vice versa. So I guess the number of errors could vary.

It's also true that for now there would be support to run in parallel just the system tests. The other type of tests would be running sequentially.

> Maybe another mid-ground option is to install all pipelines included in the package in the Setup step, and remove them on TearDown, but then we would be installing all pipelines, even the ones that are not tested.

Do you mean installing and removing them in the `SetupRunner` and `TearDownRunner` methods? so those are just performed once

Currently, those methods just return `error` in the interface, if we want to do that probably they should also return `[]testrunner.Result` , to return errors if any in those steps.